### PR TITLE
Milestone 4: Bookstore Styling

### PR DESCRIPTION
**Pull request by tresorsawasawa**
Tasks filled in this milestone:
- [ ] Add the layout styling from Bookstore design in [Zeplin](https://zeplin.io/) template.

### CODE REVIEW

File: src/scss/base/_utilities.scss [hunk 1]
```diff
@@ -22,4 +22,57 @@ a {
...
+
+.big-line {
+  width: 0.125rem;
+  height: 4.375rem;
+  margin: 1.125rem 0 0 0;
+  border: solid 1px $gary-line;
+}
+
+// .links li a { color: red; }
+
```
Comment (Cathella) [inline src/scss/base/_utilities.scss]: - [x] I suggest that you remove this commented out line of code if its no longer needed. This helps in keeping the code-base clean and readable.

File: src/scss/base/_variables.scss [hunk 2]
```diff
@@ -4,3 +4,8 @@ $blue-prim-color: #0290ff;
 $blue-secondary-color: #4386bf;
 $black-prim-color: #1d1919;
 $gray-color: #888;
+$gary-line: #e8e8e8;
+$p: 80%;
+$v: calc(((18 / 5) * $p - 90) * 1deg);
```
Comment (Cathella) [inline src/scss/base/_variables.scss]: After the application compiling in my local environment, this variable is frowned upon and cannot allow the project to run further. It only compiles after I comment it out. Below is the screenshot of the error 👇🏻 

image

- [x] Please kindly fix this error

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @tresorsawasawa 👋🏻, 

Your project looks amazing because you have done a really nice job!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct Github Flow.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello @tresorsawasawa, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**